src/__tests__/index.test.ts

```diff
@@ -420,6 +420,13 @@ const vectors: Array<TestVector> = [
       { text: '0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275', hex: '67316300f17f063085ca8bca4bd3f7a5a3c66275' },
     ],
   },
+  {
+    name: 'GXC',
+    coinType: 53890,
+    passingVectors: [
+      { text: 'GXC6UKk9URcsCuGxLuRDqEuGzAqDkgKbG8AuWXFXsyzc2r9z7A1kw', hex: '02d085655f8060a79a4b12b14e442b8a554ba867bdadce3c2dc39e1a42a01827c0' },
```

Comment:
Can you let me know where did you find the test text/hex and coinType? According to SLIP44, the coin type of GXC is 2303, not 53890 https://github.com/satoshilabs/slips/blob/master/slip-0044.md